raiden/ui/app.py

```diff
@@ -129,15 +129,15 @@ def get_account_and_private_key(
 
 
 def get_smart_contracts_start_at(network_id: ChainID) -> BlockNumber:
-    if network_id == Networks.MAINNET:
+    if network_id == Networks.MAINNET.value:
```

Comment:
These were all wrong, the `.value` is the `int`, the constant is an enum wrapper.